lib/Steps/Integrations/ReactNative.ts

```diff
@@ -243,9 +243,10 @@ export class ReactNative extends MobileProject {
         'export SENTRY_PROPERTIES=sentry.properties\n' +
         'export EXTRA_PACKAGER_ARGS="--sourcemap-output $DERIVED_FILE_DIR/main.jsbundle.map"\n' +
         code.replace(
-          /^.*?\/(packager|scripts)\/react-native-xcode\.sh\s*/m,
-          (match: any) =>
-            `../node_modules/@sentry/cli/bin/sentry-cli react-native xcode ${match}`,
+          '$REACT_NATIVE_XCODE',
+          () =>
+            // eslint-disable-next-line no-useless-escape
+            '\\\"../node_modules/@sentry/cli/bin/sentry-cli react-native xcode $REACT_NATIVE_XCODE\\\"',
```

Comment:
You can try to use backticks instead, it should make this eslint rule redundant and reduce number of escapes needed

Reply:
I've tried with backticks and it does not help, linter still complains, worth noting that I'm not scaping the String, I actually need 3 `\\\` -> https://docs.sentry.io/platforms/react-native/troubleshooting/#react-native-069-and-higher